src/data/objects/transfer.yaml

```diff
@@ -0,0 +1,17 @@
+name: Transfer
+prefix: transfer
+description: >
+    Transfer pool statistics
+schema:
+  billable:
+    type: Integer
+    value: 0
+    description: Amount of billable pool.
```

Comment:
This reads weird to me..   maybe "How much of the pool is billable" ?

Reply:
Or "The amount of the pool that is billable"

Reply:
I added to your suggestion slightly. Thanks.